--- aster/bibfor/comport/hujmat.F90
+++ aster/aster-14.6.0/bibfor/comport/hujmat.F90

@@ -50,7 +50,7 @@ subroutine hujmat(fami, kpg, ksp, mod, imat,&
     integer :: ndt, ndi, nvi, imat, kpg, ksp
     real(kind=8) :: materf(22, 2), tempf, nu21, nu31, nu32
     character(len=16) :: phenom
-    character(len=8) :: mod, nomc(24)
+    character(len=8) :: mod, nomc(25)
     character(len=*) :: fami
     integer :: icodre
     integer :: cerr(24)
@@ -90,6 +90,7 @@ subroutine hujmat(fami, kpg, ksp, mod, imat,&
     nomc(22)= 'RI_CYC  '
     nomc(23)= 'DILA    '
     nomc(24)= 'PTRAC   '
+    nomc(25)= 'DFUNC   '
 !af fin
 !
     materf(:,:)=0.d0


--- aster/bibfor/comport/hujpxd.F90
+++ aster/aster-14.6.0/bibfor/comport/hujpxd.F90
@@ -68,7 +68,7 @@ subroutine hujpxd(k, mater, sig, vin, prox,&
     pcref = mater(7, 2)
     pa = mater(8, 2)
     ptrac = mater(21,2)
-    pcr = pcref*exp(-beta*epsvp)
+    pcr = 100*pcref*exp(-beta*epsvp)
     m = sin(degr*phi)
 !
 !     
